File: Introduction to Programming in C++/Week3/coins.cpp
#include<bits/stdc++.h>
using namespace std;
int main() {
    int dollars;
    int cents;

    // Input dollars and cents
    cout << "# of dollars:";
    cin >> dollars;
    cout << "# of cents:";
    cin >> cents;
    cents += dollars * 100;

    // Calculate the number of coins
    int quarters = cents / 25;
    cents %= 25;
    int dimes = cents / 10;
    cents %= 10;
    int nickels = cents / 5;
    cents %= 5;
    int pennies = cents;

    // Output the ans
    cout << "The coins are " << quarters << " quarters, " << dimes << " dimes, " << nickels << " nickels and " << pennies << " pennies" << endl;
    return 0;
}

File: Basics of Computing and Programming/coins.py
def main():
    print("Please enter the amount of money to convert:")
    dollars = int(input("# of dollars: "))
    cents = int(input("# of cents: "))
    total = dollars * 100 + cents
    print(f"The coins are {total // 25} quarters, {total % 25 // 10} dimes, {total % 25 % 10 // 5} nickels and {total % 25 % 10 % 5} pennies")

if __name__ == "__main__":
    main()